src/Microsoft.VisualStudio.SDK.Analyzers.CodeFixes/build/AdditionalFiles/BannedSymbols.txt

```diff
@@ -1,2 +1,2 @@
 ﻿P:System.Net.ServicePointManager.SecurityProtocol; Modifying this would affect Visual Studio's process wide networking state.
-M:Microsoft.VisualStudio.Shell.Interop.IVsOutputWindowPane.OutputString(System.String); Use OutputStringThreadSafe instead, which is not any more free-threaded but runs a more limited message pump. Using IVsOutputWindowPaneNoPump.OutputStringNoPump is even better.
+M:Microsoft.VisualStudio.Shell.Interop.IVsOutputWindowPane.OutputString(System.String); Use OutputStringThreadSafe instead, which is not any more free-threaded but runs a more limited message pump. Even better, consider using IVsOutputWindowPaneNoPump.OutputStringNoPump if the interface is implemented on the target object. Or try OutputWindowTextWriter which is a free-threaded, more modern option.
```

Comment:
> which is not any more free-threaded

Weird grammar - maybe prefer "is also not free-threaded?" Wonder if this whole recommendation should rephrase to put the best option first:

"Prefer OutputWindowTextWriter for a free-threaded alternative. If that is not feasible, prefer IVsOutputWindowPaneNoPump.OutputStringNoPump (free-threaded) or OutputStringThreadSafe (not free-threaded)."

Reply:
I like your rewording. Thanks.

Reply:
I like your rewording. Thanks.